src/applications/personalization/dashboard/components/benefit-payments/PaymentsCard.jsx

```diff
@@ -3,50 +3,72 @@ import PropTypes from 'prop-types';
 import { format } from 'date-fns';
 import CTALink from '../CTALink';
 import recordEvent from '~/platform/monitoring/record-event';
+import { Toggler } from '~/platform/utilities/feature-toggles';
 
 export const PaymentsCard = ({ lastPayment }) => {
   const paymentDate = new Date(lastPayment.payCheckDt);
-  return (
-    <div className="vads-u-display--flex vads-u-flex-direction--column large-screen:vads-u-flex--1 vads-u-margin-bottom--2p5">
-      <div
-        className="vads-u-background-color--gray-lightest vads-u-padding-y--2p5 vads-u-padding-x--2p5"
-        data-testid="payment-card"
+
+  const content = (

```

Comment:
Something to consider is making this it's own component declared outside of the PaymentsCard component. I think it's generally considered best practice to declare them outside so that it isn't recreated on each render. Granted this component isn't rendered often or super dynamic, so I could go either way on this and certainly non-blocking.

Reply:
that's a good point, definitely will take that into consideration when building new components. i was also thinking about how this update is temporary with the feature toggle.